index.js

```diff
@@ -16,9 +16,23 @@ const manageRecipes = async () => {
     console.log(`Connected to the database: "${dbConnection.connection.name}"`);
 
     // Before adding any recipes to the database, let's remove all existing ones
-    await Recipe.deleteMany();
+    //await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+    //Recipe.insertMany(data)
+
+    //Print all the titles
+    //let title = await Recipe.find({}, {"title":1})
+    //console.log(title)
+
+    //Update Recipe
+    /* await Recipe.findOneAndUpdate({title: "Rigatoni alla Genovese"},{duration: 100})
```

Comment:
If you wanted, you could've set the `new` option to `true` to return the document after the update was applied.

`await Recipe.findOneAndUpdate({title: "Rigatoni alla Genovese"},{duration: 100}, {new: true})`